game/views.py

```
@@ -41,8 +41,20 @@ def index(request):
-    user = request.user if request.user.is_authenticated else None
-    GameResult.objects.create(user=user, mode=mode, winner=winner, end_reason=reason, player_color=player_color)
+    try:
+        user = request.user if request.user.is_authenticated else None
+        GameResult.objects.create(
+            user=user,
+            mode=mode,
+            winner=winner,
+            end_reason=reason,
+            player_color=player_color
+        )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)